lua/core/autopairs.lua

```diff
@@ -1,51 +1,60 @@
--- if not package.loaded['nvim-autopairs'] then
---   return
--- end
-local status_ok, _ = pcall(require, "nvim-autopairs")
-if not status_ok then
-  return
+local M = {}
+
+M.config = function()
+  lvim.builtin.autopairs = {
+    active = true,
+  }
 end
-local npairs = require "nvim-autopairs"
-local Rule = require "nvim-autopairs.rule"
 
--- skip it, if you use another global object
-_G.MUtils = {}
+M.setup = function()
+  local status_ok, _ = pcall(require, "nvim-autopairs")
+  if not status_ok then
+    return
+  end
+  local npairs = require "nvim-autopairs"
+  local Rule = require "nvim-autopairs.rule"
 
-vim.g.completion_confirm_key = ""
-MUtils.completion_confirm = function()
-  if vim.fn.pumvisible() ~= 0 then
-    if vim.fn.complete_info()["selected"] ~= -1 then
-      return vim.fn["compe#confirm"](npairs.esc "<cr>")
+  -- skip it, if you use another global object
+  _G.MUtils = {}
+
+  vim.g.completion_confirm_key = ""
+  MUtils.completion_confirm = function()
+    if vim.fn.pumvisible() ~= 0 then
+      if vim.fn.complete_info()["selected"] ~= -1 then
+        return vim.fn["compe#confirm"](npairs.esc "<cr>")
+      else
```

Comment:
I know it's not your code, but while at it, do you mind removing the else blocks after `return`
``` lua
      if vim.fn.complete_info()["selected"] ~= -1 then
        return vim.fn["compe#confirm"](npairs.esc "<cr>")
      end
      return npairs.esc "<cr>"
```

The same applies for L:29

Reply:
I am not sure why, but done.